build/Build.Steps.cs

```diff
@@ -42,9 +44,17 @@ partial class Build
             : new[] { MSBuildTargetPlatform.x86 };
 
     private static readonly IEnumerable<TargetFramework> TargetFrameworks = new[]
+    {
+       TargetFramework.NET8_0,
+       TargetFramework.NET462,
```

Comment:
`TargetFramework.NET462` should this remain here? Becomes quite confusing between `TargetFrameworks` vs `TargetFrameworksNetFx `

Reply:
`TargetFrameworksForNetFxRedirections`?

Reply:
`TargetFrameworksForNetFxPacking`?